samples/project-geometry/src/main/java/com/esri/arcgismaps/sample/projectgeometry/components/ProjectGeometryViewModel.kt

```diff
@@ -35,22 +35,31 @@ import com.arcgismaps.toolkit.geoviewcompose.MapViewProxy
 import com.esri.arcgismaps.sample.projectgeometry.R
 import com.esri.arcgismaps.sample.sampleslib.components.MessageDialogViewModel
 import kotlinx.coroutines.flow.MutableStateFlow
-import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
 import kotlinx.coroutines.launch
-import java.util.Locale
+
+data class PointProjection (val original: Point, val projection: Point)
 
 class ProjectGeometryViewModel(val app: Application) : AndroidViewModel(app) {
     // create a map with a navigation night basemap style
     val arcGISMap = ArcGISMap(BasemapStyle.ArcGISStreetsNight).apply {
         // set the default viewpoint to Redlands,CA
-        initialViewpoint = Viewpoint(34.058, -117.195, 5e4)
+        initialViewpoint = Viewpoint(
+            latitude = 34.058,
+            longitude = -117.195,
+            scale = 5e4
+        )
     }
     // create a MapViewProxy to interact with the MapView
     val mapViewProxy = MapViewProxy()
 
     // Create a message dialog view model for handling error messages
     val messageDialogVM = MessageDialogViewModel()
 
+    // state flow of a point and its projection for presentation in UI
+    private val _pointProjectionFlow = MutableStateFlow<PointProjection?>(null)
+    val pointFlow = _pointProjectionFlow.asStateFlow()
```

Comment:
A `StateFlow` and the private `MutableStateFlow` backing it should have the same name (apart from the `_` beforehand)
```suggestion
    private val _pointProjectionFlow = MutableStateFlow<PointProjection?>(null)
    val pointProjectionFlow = _pointProjectionFlow.asStateFlow()
```